Sources/Networking/Responses/OfferingsResponse.swift

```diff
@@ -29,6 +29,8 @@ struct OfferingsResponse {
         let identifier: String
         let description: String
         let packages: [Package]
+        @IgnoreDecodeErrors<PaywallData?>
+        var paywall: PaywallData?
```

Comment:
Important that we don't fail to load the offering if this fails. The error will get logged with #2778

Reply:
+1000. At least for now

Reply:
we should fail at the Paywalls SDK level though

Reply:
This is what I'm thinking:

```swift
if let paywall = offering.paywall {
  // The paywall is required
  PaywallView(offering: offering, paywall: paywall)
} else {
  // You gotta handle it and do something yourself.
}
```

With the `public` `PaywallData` constructor, users could to this:

```swift
} else {
  // This is hardcoded in the app
  let simplePaywall = PaywallData(
      template: .simple
      config: ...
  )
  PaywallView(offering: offering, paywall: simplePaywall)
}
```